### PR TITLE
adds my answer to exercise 1

### PR DESCRIPTION
**Pull request by katyhuff**
I added my answer just at the bottom of the file. Hopefully that was the intention.

### CODE REVIEW

File: _exercises/02-01-putting-your-vision-into-words.md [hunk 1]
```diff
@@ -18,3 +18,14 @@ go. Try one of the following as a guide. Just fill in the blanks:
...
 
 *   I see a (fill in the blank with whatever it is you are trying to change) where… [____________________]
+
+
+## Katy Huff
+
+I have two because I'm indecisive.
+
+- I see a group of nuclear collaborators (graduate advisees, peer professors, 
+  laboratory scientists) that advance nuclear engineering sustainability.
```
Comment (gvwilson) [inline _exercises/02-01-putting-your-vision-into-words.md]: Would this group include people outside the nuclear field (environmental activists, communities that want cheaper/more reliable power, etc.), or would it be smaller with more knowledgeable/committed membership reaching out to the community at large?

File: _exercises/02-01-putting-your-vision-into-words.md [hunk 2]
```diff
@@ -18,3 +18,14 @@ go. Try one of the following as a guide. Just fill in the blanks:
...
+- I see a self-sustaining proliferation of hyper-local communities of peer 
+  learning who effectively share skills for best practices in scientific 
+  computing.
```
Comment (gvwilson) [inline _exercises/02-01-putting-your-vision-into-words.md]: I presume "hyper-local" means campus by campus?  What about department by department?  Do you envisage e.g. ecologists and physicsts at the same campus being in the same group, or different groups for different disciplines?

Comment (katyhuff) [inline _exercises/02-01-putting-your-vision-into-words.md]: Yeah. I'm just talking about THW, so I envision 1 per university. I think it benefits from the diversity.

File: _exercises/02-01-putting-your-vision-into-words.md (hunk 1 above)
Comment (katyhuff) [inline _exercises/02-01-putting-your-vision-into-words.md]: I'm just talking about my immediate research collaborators and students at this point, but it already has connections to related groups as you mentioned (e.g. environmental activists, national gov't, think tanks, local communities, workers unions) through driving to retain our current nuclear plants.

**Pull request merged**